### ISSUE TITLE
Do I need a Softmax at the end of a pretrained network?

### ISSUE THREAD

**Issue by mrclasalvia** [facebookresearch/pytorchvideo#81]
When I load a pretrained network and I specify the number of classes, I correctly receive a tensor having 4 dimensions. However, I then compare this with the labels as suggested in the tutorial. Do I need something that retrieves the index of the most probable class or it is handled automatically by lightning? 

Thanks

**Comment by kalyanvasudev** [comment]
Hi, please checkout our inference using pretrained model tutorial for more details - https://pytorchvideo.org/docs/tutorial_torchhub_inference

As already mentioned in our tutorial, you need softmax at the top.
